### ISSUE TITLE
Latest version not working?

### ISSUE THREAD

**Issue by netbrain** [hybridgroup/gocv#458]
Seems the latest version of opencv is failing with gocv.

## Description
```
go run ./cmd/version/main.go
# command-line-arguments
/usr/local/go/pkg/tool/linux_amd64/link: running g++ failed: exit status 1
/usr/bin/ld: /tmp/go-link-360343100/000016.o: in function `Net_ReadNet':
/home/netbrain/dev/gocv/dnn.cpp:4: undefined reference to `cv::dnn::dnn4_v20181221::readNet(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
...
```

## Steps to Reproduce
1.  fetch latest dev branch
2.  make deps download build_cuda sudo_install verify

## Your Environment
$ cat /etc/issue && uname -r
Debian GNU/Linux 10 \n \l
4.19.0-5-amd64

* GoCV version used: latest dev (a0e3403cfa575254bfb43d0f9999c6cfd2ea76c7)
* Go version: 1.12
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`? no

**Comment by deadprogram** [comment]
You might need to run `go clean --cache` before compiling if you are "upgrading in place".

**Comment by netbrain** [comment]
Yes, you are correct. This solved the issue. Maybe this should be included as a step in the makefile?

**Comment by deadprogram** [comment]
Very good idea. Would you like to submit a PR for this?

**Comment by netbrain** [comment]
Sure thing.!

**Comment by deadprogram** [comment]
This has now been released with v0.21.0 thanks again! Now closing.
